### ISSUE TITLE
Add daily boosted creature + boss endpoints

### ISSUE THREAD

**Issue by mathiasbynens** [TibiaData/tibiadata-api-go#163]
This data could be scraped from https://www.tibia.com/library/?subtopic=creatures and https://www.tibia.com/library/?subtopic=boostablebosses, respectively, or in a single request by visiting most pages on tibia.com e.g. `https://www.tibia.com/news/?subtopic=latestnews` where the HTML contains something like:

```html
<img id="Pedestal" src="https://static.tibia.com/images/global/header/pedestal.gif" alt="Monster Pedestal Box" /><br><img id="Monster" title="Today's boosted creature: Gozzler" src="https://static.tibia.com/images/global/header/monsters/gozzler.gif" onClick="window.location = 'https://www.tibia.com/library/?subtopic=creatures';" alt="Boosted Creature" /><img id="Boss" title="Today's boosted boss: Srezz Yellow Eyes" src="https://static.tibia.com/images/global/header/monsters/srezz.gif" onClick="window.location = 'https://www.tibia.com/library/?subtopic=boostablebosses';" alt="Boosted Boss" />
```

**Comment by tobiasehlert** [comment]
hi @mathiasbynens,

There is an implementation for both boosted creature and boosted boss available already.

Boosted boss you can find in endpoint `/v3/boostablebosses`
[Link to docs](https://docs.tibiadata.com/#get-/v3/boostablebosses)

Boosted creature you can find in endpoint `/v3/creatures`
[Link to docs](https://docs.tibiadata.com/#get-/v3/creatures)

Kind regards,
Tobias

**Comment by mathiasbynens** [comment]
Oh, I see. It wasn't obvious to me from reading the docs that those two endpoints include that information.

Have you considered something like `/v3/boosted/boss` and `/v3/boosted/creature`, returning just that particular bit of information? (Or perhaps `/v3/boosted` containing both the boosted boss and creature?)

**Comment by tobiasehlert** [comment]
@mathiasbynens, I don't really see the point to expose those two specific values in a separate endpoint.

Maybe I could add that in the upcoming "generic" endpoint that will show twitch, youtube, booyah status and add a boosted section there as well. Or what you think?

**Comment by mathiasbynens** [comment]
No worries. Feel free to close this issue!

Here's what I was thinking when I filed this: when I saw the endpoint named `boostablebosses`, I expected it to return the list of boostable bosses (only). When I saw the endpoint named `creatures`, I expected it to return the list of creatures (only). The detail that these endpoints also include the daily boosted boss/creature wasn't intuitive to me. That's it :) I'm more than happy to use the existing endpoints.
